.github/workflows/web-app-deployer.yml

```diff
@@ -211,10 +209,10 @@ jobs:
 
       - name: Select Firebase Project
         working-directory: web-app
-        run: npx firebase use ${{  env.FIREBASE_PROJECT }}
+        run: npx firebase use ${{  inputs.FIREBASE_PROJECT }}
 
-      - name: Deploy to Firebase Hosting (${{ env.FIREBASE_PROJECT }})
-        if: ${{ env.FIREBASE_PROJECT != 'dev' }}
+      - name: Deploy to Firebase Hosting (${{ inputs.FIREBASE_PROJECT }})
+        if: ${{ github.event_name != 'pull_request' }}
```

Comment:
This will always happen since you've removed the `pull_request` trigger, no?

Reply:
Fixed, added an additional GH action